google/api_core/future/polling.py

```diff
@@ -122,7 +124,7 @@ def result(self, timeout=None):
             google.api_core.GoogleAPICallError: If the operation errors or if
                 the timeout is reached before the operation completes.
         """
-        self._blocking_poll(timeout=timeout)
+        self._blocking_poll(timeout=timeout, retry=retry)
```

Comment:
Even though `_blocking_poll` is private, it seems we do override it in some classes. We'll need to guard with a similar `**kwargs` check.

Reply:
Done